scripts/generic/config.ts

```diff
@@ -9,14 +9,14 @@
 // type is fontsource-specific e.g. google, league, other. It should be left as "other" unless otherwise specified.
 
 const config = {
-  fontId: "",
-  fontName: "",
+  fontId: "bluu-next",
```

Comment:
Just need to clear up the config file and we can merge this 👍

Reply:
Sorry about that. Committed new changes.